apps/meteor/server/startup/migrations/v285.ts

```diff
@@ -0,0 +1,35 @@
+import { Messages } from '@rocket.chat/models';
```

Comment:
@sampaiodiego , can you help us to validate the migration?

My question is: do we need 2 queries for doing that? I think we can get everything on the updateMany call 👀